crates/stackable-versioned-macros/Cargo.toml

```diff
@@ -6,6 +6,9 @@ license.workspace = true
 edition.workspace = true
 repository.workspace = true
 
+[package.metadata.cargo-udeps.ignore]
+development = ["schemars", "serde_yaml"]
+
```

Comment:
Can we add an explanation why they are ignored, for posterity?
Will there be a time they can be removed from the list?

Reply:
Ah yes, I can add such a comment.

> Will there be a time they can be removed from the list?

They can eventually be removed once cargo-udeps supports checking dev-dependencies (which are used in tests).

Reply:
Added in 9529a50